compiler+runtime/src/cpp/jank/runtime/context.cpp

```diff
@@ -282,7 +280,11 @@ namespace jank::runtime
       absolute_module = module::nest_module(ns->to_string(), module);
     }
 
-    binding_scope const preserve{ *this };
+    binding_scope const preserve{ *this,
+                                  obj::persistent_hash_map::create_unique(
+                                    std::make_pair(current_ns_var, ns),
+                                    std::make_pair(current_module_var,
+                                                   make_box(absolute_module))) };
```

Comment:
Let's explain what's going on here. Remember how this wasn't clear to you when you saw it in Clojure's code.